### PR TITLE
Removed literals for bundle names

### PR DESCRIPTION
**Pull request by wouterj**
_Note: Most of it was automated and I didn't review it closely yet (since I didn't know the general opinion on this topic), so there can be some faulty changes._

I always like to use as less literals as possible, as I think it makes sentences harder to understand. That's why I strongly believe that literals should exclusively be used for code. As a bundle name is not code, I propose to remove all literals around bundle names.

| Q | A |
| --- | --- |
| Doc fix? | yes |
| New docs? | nope |
| Applies to | all |
| Fixed tickets | - |

### CODE REVIEW

**Comment by stof** [comment]
I tend to agree with you

File: best_practices/business-logic.rst [hunk 1]
```diff
@@ -45,7 +45,7 @@ and put things there:
 
 .. tip::
 
-    The recommended approach of using the ``AppBundle`` directory is for
+    The recommended approach of using the AppBundle directory is for
```
Comment (xabbuh) [inline best_practices/business-logic.rst]: AppBundle refers to a directory name.

File: best_practices/controllers.rst [hunk 2]
```diff
@@ -13,8 +13,8 @@ into a service.
 
 .. best-practice::
 
-    Make your controller extend the ``FrameworkBundle`` base Controller and
-    use annotations to configure routing, caching and security whenever possible.
+    Make your controller extend the FrameworkBundle base Controller and use
```
Comment (xabbuh) [inline best_practices/controllers.rst]: You can also lowercase Controller (or put it in literals if we use it as the class name here).

**Comment by xabbuh** [comment]
:+1: (besides the two minor things I commented on)

File: best_practices/business-logic.rst (hunk 1 above)
Comment (WouterJ) [inline best_practices/business-logic.rst]: Good catch! I've changed it to `AppBundle/` to make it even more clear

**Comment by WouterJ** [comment]
@weaverryan before merging, I first want your opinion on this :)

File: best_practices/business-logic.rst (hunk 1 above)
Comment (xabbuh) [inline best_practices/business-logic.rst]: :+1:

File: book/service_container.rst [hunk 3]
```diff
@@ -423,10 +423,9 @@ configuration.
 The ``imports`` directive allows your application to include service container
 configuration resources from any other location (most commonly from bundles).
 The ``resource`` location, for files, is the absolute path to the resource
-file. The special ``@AcmeHello`` syntax resolves the directory path of
-the ``AcmeHelloBundle`` bundle. This helps you specify the path to the resource
-without worrying later if you move the ``AcmeHelloBundle`` to a different
-directory.
+file. The special ``@AcmeHello`` syntax resolves the directory path of the
```
Comment (weaverryan) [inline book/service_container.rst]: This is (and was) a typo, right? Should be `@AcmeHelloBundle`

**Comment by weaverryan** [comment]
@WouterJ Hmm, I like your point - the literals make things stand out, which can be bad for reading. So I definitely support this! In some cases, the bundle is referring to a directory, and in that case it should still have the literals. But I noticed that you did exactly this in your PR, so nice work :).

File: book/service_container.rst (hunk 3 above)
Comment (weaverryan) [inline book/service_container.rst]: Fixed at sha: b624100a6d886905d4c63c74949ce03a2d805eb4 - someone tackle me if I'm missing something :)

**Pull request merged**